aci/aci.go

```diff
@@ -271,10 +271,10 @@ func (c *Client) RefreshDeadline() time.Time {
 
 func tlsConfig() *tls.Config {
 	return &tls.Config{
-		CipherSuites:             []uint16{tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA, tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA},
+		CipherSuites:             []uint16{tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA, tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA, tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256, tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384},
 		PreferServerCipherSuites: true,
 		InsecureSkipVerify:       true,
-		MaxVersion:               tls.VersionTLS11,
+		MaxVersion:               tls.VersionTLS12,
```

Comment:
Any reason to limit only to TLS1.2? Shouldn't the `MaxVersion` be the latest stable available (which is TLS1.3).

Reply:
At the time the code was first put in place, APIC only worked well with Go for that specific combination of ciphers and TLS11.
Allowing other options would break the negotiation then.
Don't know the state of things nowdays.